### PR TITLE
Update: Switch from master to main branch

### PR DESCRIPTION
**Pull request by iokpala**
# Summary | Résumé

Going forward, we are adopting main as our default branch,

This commit updates the workflows YAML files, replacing master with  main branch

### CODE REVIEW

**Comment by jimleroyer** [comment]
Spoke with @iokpala who started on the effort. This is a low hanging fruit task to do in-between tasks, while waiting for new ones.

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -42,7 +43,7 @@ jobs:
         cp -f .env.example .env
 
     - name: Checks for new endpoints against AWS WAF rules
-      uses: cds-snc/notification-utils/.github/actions/waffles@master
+      uses: cds-snc/notification-utils/.github/actions/waffles@main
```
Comment (sastels) [inline .github/workflows/push.yml:46]: This probably needs to stay at master if we're referencing `notification-utils`? Thoughts @jimleroyer ?

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -28,9 +26,12 @@ jobs:
...
     - name: Run tests
       run: make test
-
+    - name: Get python version
+      run: |
+        python_version=$(python  -V | cut -d' ' -f2)
+        echo "python_version=${python_version}" >> $GITHUB_ENV
     - name: Copy site-packages in workspace
-      working-directory: /opt/hostedtoolcache/Python/3.9.9/x64/lib/python3.9/
+      working-directory: /opt/hostedtoolcache/Python/${{ env.python_version }}/x64/lib/python3.9/
```
Comment (sastels) [inline .github/workflows/push.yml:34]: This is a good idea!

File: .github/workflows/push.yml (hunk 1 above)
Comment (iokpala) [reply .github/workflows/push.yml:46]: I have created `main` https://github.com/cds-snc/notification-utils/tree/main

File: .github/workflows/push.yml (hunk 2 above)
Comment (iokpala) [reply .github/workflows/push.yml:34]: It failed possible due to change in python version.. see https://github.com/cds-snc/notification-document-download-api/runs/5205000366?check_suite_focus=true

File: .github/workflows/push.yml (hunk 1 above)
Comment (jimleroyer) [reply .github/workflows/push.yml:46]: I guess it will be good then. 👍

**Pull request merged**